modules/core/AgsCore.swift

```diff
@@ -1,25 +1,44 @@
+import AeroGearHttp
 import Foundation
+import XCGLogger
 
 /**
  * AeroGear Core interface to be used directly by SDK services
  */
 public class AgsCore {
     public static let instance = AgsCore()
 
-    let config: Config
+    let config: ServiceConfig
 
     public init() {
```

Comment:
Nice singleton :)

Reply:
It's not singleton :P

Reply:
So please name it as default instead of instance